tests/netperf_test.py

```diff
@@ -103,66 +105,47 @@ def run_netperf_tcp(device, run_time, pkt_size, direction="up"):
         cmd = "netperf -H %s -c -C -l %s -t TCP_MAERTS -- -m %s -M %s -D" % (wan.gw, run_time, pkt_size, pkt_size)
     device.sendline(cmd)
     device.expect('TEST.*\r\n')
-    device.expect(device.prompt, timeout=run_time+4)
+    device.expect(device.prompt, timeout=run_time+6)
 
-class Netperf_UpTCP256(rootfs_boot.RootFSBootTest):
+class Netperf_UpTCP256(NetperfTest):
     '''Netperf upload throughput (TCP, packets size 256 Bytes).'''
 
     def runTest(self):
-        lib.common.test_msg("Starting netserver on wan...")
-        wan.sendline('/usr/bin/netserver')
-        wan.expect("Starting netserver with host")
+        super(Netperf_UpTCP256, self).start_netserver(wan)
         run_netperf_tcp(device=lan, run_time=15, pkt_size=256)
-        wan.sendline("kill -9 `pidof netserver`")
-        wan.expect(prompt)
+        super(Netperf_UpTCP256, self).kill_netserver(wan)
 
-class Netperf_UpTCP512(rootfs_boot.RootFSBootTest):
+class Netperf_UpTCP512(NetperfTest):
     '''Netperf upload throughput (TCP, packets size 512 Bytes).'''
     def runTest(self):
-        lib.common.test_msg("Starting netserver on wan...")
-        wan.sendline('/usr/bin/netserver')
-        wan.expect("Starting netserver with host")
+        super(Netperf_UpTCP512, self).start_netserver(wan)
         run_netperf_tcp(device=lan, run_time=15, pkt_size=512)
```

Comment:
While these now inherit properly, the are not using the default implementation on runTest() on the parent class. That should probably be fixed at the same time.

Also, you should break this into a separate change from the start/stop server functions.